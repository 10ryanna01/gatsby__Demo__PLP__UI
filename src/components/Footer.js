import React from "react"
import { Link } from "gatsby" 
import "./styles/index.less"

const Footer = () => (
  
  <footer className={`footer`}>
    <div>
      <h3 className={`footer__title`}>You made past the infinate scroll #yay </h3>
        
        <div className={`footer__links`}> 
        <Link to="/">
            info
        </Link>
        <Link to="/">
         marketing 
        </Link>
        <Link to="/">
         support 
        </Link>
        </div>
    </div>
  </footer>
   
)

 
export default Footer

