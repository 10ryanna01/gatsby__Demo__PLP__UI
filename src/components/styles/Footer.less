.footer {
  background: @brandOldPrice;
  text-align: center;
  font-family: @brandFontFamily;
  .footer,
  &__title {
    font-size: @copyFineprint;
    color: @brandFinePrint;

    font-weight: 300;
    cursor: pointer;
    padding-bottom: 1rem;
    padding-top: 0.8rem;
    font-weight: 300;
    background: linear-gradient(
      to bottom,
      @brandSecondary 100%,
      @brandSecondary 0%
    );
    background-position: 100 0%;
    background-repeat: repeat-x;
    background-size: 8px 4px;
    color: @colorNeutralBright;
    text-decoration: none;
    transition: background-size 0.4s;

    &:hover {
      background-size: 4px 50px;
      color: @colorNeutralDark;
    }

    &__links a {
      font-size: @copyFineprint;
      color: @colorNeutralBright;
      padding: 1rem;
      text-decoration: none;
      font-weight: 300;
    }
  }
}
