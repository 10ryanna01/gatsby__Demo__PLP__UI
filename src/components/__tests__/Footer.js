import React from "react"
import renderer from "react-test-renderer"
import { Link } from "gatsby"
import Footer from "../Footer"


describe("Footer", () => {
    it("renders correctly as expected", () => {
        const tree = renderer
            .create(<Footer className={`footer`}>
                <div>
                    <h3 className={`footer__title`}>You made past the infinate scroll #yay </h3>

                    <div className={`footer__links`}>
                        <Link to="/">
                            info
          </Link>
                        <Link to="/">
                            marketing
          </Link>
                        <Link to="/">
                            support
          </Link>
                    </div>
                </div>
            </Footer>

            )
            .toJSON()
        expect(tree).toMatchSnapshot()
    })
})