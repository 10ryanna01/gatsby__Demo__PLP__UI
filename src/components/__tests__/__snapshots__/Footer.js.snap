// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Footer renders correctly as expected 1`] = `
<footer
  className="footer"
>
  <div>
    <h3
      className="footer__title"
    >
      You made past the infinate scroll #yay 
    </h3>
    <div
      className="footer__links"
    >
      <a
        href="/"
      >
        info
      </a>
      <a
        href="/"
      >
        marketing
      </a>
      <a
        href="/"
      >
        support
      </a>
    </div>
  </div>
</footer>
`;
